src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -124,6 +124,8 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 	readonly onDidChangeCellState = this._onDidChangeCellState.event;
 	private readonly _onDidChangeViewCells = this._register(new Emitter<INotebookViewCellsUpdateEvent>());
 	readonly onDidChangeViewCells: Event<INotebookViewCellsUpdateEvent> = this._onDidChangeViewCells.event;
+	private readonly _onWillChangeModel = this._register(new Emitter<NotebookTextModel | undefined>());
+	readonly onWillChangeModel: Event<NotebookTextModel | undefined> = this._onWillChangeModel.event;
```

Comment:
I'm not seeing where this is used?

Reply:
it's used here https://github.com/microsoft/vscode/blob/99c2baa86bb481fafe66229285f260d8d7ce5d79/src/vs/workbench/contrib/search/browser/searchModel.ts#L1416